readthedocs/builds/models.py

```diff
@@ -360,6 +358,9 @@ def get_github_url(
             return ''
         repo = repo.rstrip('/')
 
+        if not filename:
+            source_suffix = ''
```

Comment:
This could use a comment, otherwise I don't understand what it's doing. Is this us "only linking to the docdir"?